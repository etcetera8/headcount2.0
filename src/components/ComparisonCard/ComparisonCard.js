import React from 'react';

const ComparisonCard = () => {

  return (
    <article>
      <h3>{d1Avg}</h3>
    </article>

  )


}

export default ComparisonCard;