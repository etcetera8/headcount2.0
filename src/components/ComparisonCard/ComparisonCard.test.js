import React from 'react';
import ComparisonCard from './ComparisonCard';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'

it('should render', () => {
  const mockData = {COLORADO: 0.53, 'ACADEMY 20': 0.407, compared: 1.302}
  const wrapper = shallow(<ComparisonCard comparedObject={mockData} />)

  expect(wrapper.find('article.compare-card').exists()).toEqual(true);
})

it('should match the snapshot', () => {
  const mockData = {COLORADO: 0.53, 'ACADEMY 20': 0.407, compared: 1.302}
  const wrapper = shallow(<ComparisonCard comparedObject={mockData} />)

  expect(wrapper).toMatchSnapshot();
})