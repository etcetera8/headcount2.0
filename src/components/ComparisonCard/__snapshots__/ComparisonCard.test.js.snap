// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ComparisonCard
    comparedObject={
        Object {
            "ACADEMY 20": 0.407,
            "COLORADO": 0.53,
            "compared": 1.302,
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3
          className="top"
>
          COLORADO
</h3>,
        <h4>
          0.53
           avg
</h4>,
        <h1>
          1.302
</h1>,
        <h4>
          0.407
           avg
</h4>,
        <h3
          className="bottom"
>
          ACADEMY 20
</h3>,
      ],
      "className": "compare-card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "COLORADO",
          "className": "top",
        },
        "ref": null,
        "rendered": "COLORADO",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            0.53,
            " avg",
          ],
        },
        "ref": null,
        "rendered": Array [
          0.53,
          " avg",
        ],
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": 1.302,
        },
        "ref": null,
        "rendered": 1.302,
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            0.407,
            " avg",
          ],
        },
        "ref": null,
        "rendered": Array [
          0.407,
          " avg",
        ],
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "ACADEMY 20",
          "className": "bottom",
        },
        "ref": null,
        "rendered": "ACADEMY 20",
        "type": "h3",
      },
    ],
    "type": "article",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3
            className="top"
>
            COLORADO
</h3>,
          <h4>
            0.53
             avg
</h4>,
          <h1>
            1.302
</h1>,
          <h4>
            0.407
             avg
</h4>,
          <h3
            className="bottom"
>
            ACADEMY 20
</h3>,
        ],
        "className": "compare-card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "COLORADO",
            "className": "top",
          },
          "ref": null,
          "rendered": "COLORADO",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              0.53,
              " avg",
            ],
          },
          "ref": null,
          "rendered": Array [
            0.53,
            " avg",
          ],
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": 1.302,
          },
          "ref": null,
          "rendered": 1.302,
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              0.407,
              " avg",
            ],
          },
          "ref": null,
          "rendered": Array [
            0.407,
            " avg",
          ],
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "ACADEMY 20",
            "className": "bottom",
          },
          "ref": null,
          "rendered": "ACADEMY 20",
          "type": "h3",
        },
      ],
      "type": "article",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
